dist/libs/common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@groupdocs.examples.angular/common-components",
-  "version": "0.5.10",
+  "version": "0.1.16",
```

Comment:
Why version became 0.1.16 instead of 0.5.17, is it correct?

Reply:
Version will be changed after release. It is draft

Reply:
Ok, thank you.